README.md

```diff
@@ -1,25 +1,211 @@
+
+```python
+import cell_census
+with cell_census.open_soma() as census:
+   ...
+```
+
+##### Querying a slice of cell metadata.
+
+The following filters female cells of cell type "microglial cell" or "neuron", and selects the columns "assay", "cell_type" and "tissue".
+
+```python
+# Reads SOMA data frame
+cell_metadata = census["census_data"]["homo_sapiens"].obs.read(
+   value_filter = "sex == 'female' and cell_type in ['microglial cell', 'neuron']",
+   column_names = ["assay", "cell_type", "tissue"]
+)
+
+# Concatenates results to pyarrow.Table
+cell_metadata = cell_metadata.concat()
+
+# Converts to pandas.DataFrame
+cell_metadata = cell_metadata.to_pandas()
+```
+
+##### Obtaining a slice as AnnData 
+
+The following filters female cells of cell type "microglial cell" or "neuron", and selects the cell metadata columns "assay", "cell_type" and "tissue". It also filters for the genes "ENSG00000161798" and "ENSG00000188229".
+
+```python
+adata = cell_census.get_anndata(
+    census = census,
+    organism = "Homo sapiens",
+    var_value_filter = "feature_id in ['ENSG00000161798', 'ENSG00000188229']",
+    obs_value_filter = "cell_type == 'B cell' and tissue_general == 'lung' and disease == 'COVID-19'",
+    column_names = {"obs": ["assay", "cell_type", "tissue"]},
+)
+
+```
+
+##### Memory-efficient queries
+
+This example provides a demonstration to accessed the data for larger-than-memory operations using **TileDB-SOMA** operations. 
+
+First we initiate a lazy-evaluation query to access all brain and male cells from human. This query needs to be closed — `query.close()` — or used called in a context manager — `with ...`.
+
+```
+import tiledbsoma
+
+human = census["census_data"]["homo_sapiens"]
+
+query = human.axis_query(
+   measurement_name = "RNA",
+   obs_query = tiledbsoma.AxisQuery(
+      value_filter = "tissue == 'brain' and sex == 'male'"
+   )
+)
+```
+
+Now we can iterate over the matrix count, as well as the cell and gene metadata. For example, to iterate over the matrix count, we can get an iterator and perform operations for each iteration.
+
+```
+iterator = query.X("raw").tables()
+
+# Get a slice a pyarrow.Table
+raw_slice = next (iterator) 
+...
+```
+
+Alternatively.
+
+```
+for raw_slice in  query.X("raw").tables():
+   ...
+``` 
+
+And you must close the query and Cell Census after you are done.
+
+```
+query.close()
+census.close()
+```
+
+### R quick start
+
+*Coming soon*
+
+
+## Questions, feedback and issues
+
+- Questions: we encourage you to ask questions via [github issues](https://github.com/chanzuckerberg/cell-census/issues). Alternatively, for quick support you can join the C[ZI Science Community](https://join-cellxgene-users.herokuapp.com/) on Slack and join the `#cell-census-users` channel
+- Bugs: please submit a [github issue](https://github.com/chanzuckerberg/cell-census/issues). 
+- Feature requests: please submit requests using this [form](https://airtable.com/shrVV1g0d6nvBoQYu).
```

Comment:
Have we tested that the airtable is accessible to the public? 
Why aren't we using github for feature requests too? I prefer one source. airtable simply requires more work to then move issues to github.

Reply:
It has public access. As per several recommendations from the team we would like to try Airtable for the time being. Request that we want to prioritize and work on will be added to github.

Reply:
"questions" (issues) and "feature requests" (airtable) blur together. I think this fragmentation is going to create work for you. 

What functionality do you like about airtable that makes you gravitate towards it?

It's possible you could have your cake and eat it, too. See e.g.

https://github.com/satijalab/seurat/discussions

Reply:
I'm also not excited about having another tools - there are already too many sources & UIs.  What are we optimizing for here?  (i.e., "us" or "our users"?)

IMHO, feature requests should be filed in GH issues, as you would find in most other open projects.